solr/server/solr/configsets/sample_techproducts_configs/conf/solrconfig.xml

```diff
@@ -508,6 +507,23 @@
                         showItems="32" />
       -->
 
+    <!-- Feature Values Cache
+
+         Cache used by the Learning To Rank (LTR) contrib module.
+
+         You will need to set the solr.ltr.enabled system property
+         when running solr to run with ltr enabled:
+           -Dsolr.ltr.enabled=true
+
+         https://cwiki.apache.org/confluence/display/solr/Result+Reranking
+      -->
+    <cache enable="${solr.ltr.enabled:false}" name="QUERY_DOC_FV"
+           class="solr.search.LRUCache"
+           size="4096"
+           initialSize="2048"
+           autowarmCount="4096"
+           regenerator="solr.search.NoOpRegenerator" />
```

Comment:
I was thinking, since we removed the multithreading feature from the config, do we need the cache as well?